packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts

```diff
@@ -0,0 +1,173 @@
+import { taggedTemplateHelpers as utils } from '@cli/utils';
+
+type GeneratorName = 'next-wp' | 'next-drupal' | 'gatsby-wp';
+type CmsType = 'drupal' | 'wordpress';
+
+/**
+ * The header section for the starter kit READMEs
+ */
+export const readmeHeader =
+	() => /* md */ `<div style="display:flex;flex-direction:column">
+	<img src="https://raw.githubusercontent.com/pantheon-systems/decoupled-kit-js/canary/web/static/img/B_Fist-Tagline.png" height="120" style="background:#ffffff;border-radius:8px;margin:auto;display:flex;" alt="Pantheon.io logo featuring a fist capturing lighting. Pantheon™, The Platform for Extraordinary Websites.">
+	<a href="https://pantheon.io">
+		<h1 style="margin:auto;" align="center">Pantheon</h1>
```

Comment:
I don't think the h1 here should be Pantheon. I think it should instead be the name of the starter kit. Moving that outside of this component might make it easier to vary this heading.

Reply:
Makes sense to me!